### ISSUE TITLE
Inconsistent task description in rampup/reversingChanges across languages

### ISSUE THREAD

**Issue by st31ny** [pcottle/learnGitBranching#808]
I noticed that at least in some languages the task description in rampup/reversingChanges is inconsistent. Solving the task, it is clear that one commit on `local` and one commit on `pushed` needs to be reset or reverted, respectively. The English description points that out correctly: "You will revert two commits total (one per branch)."
However, at least in some other languages, this is not clear:

de_DE:
"Um diesen Level abzuschließen musst du sowohl auf `local` also auch auf `pushed` jeweils die zwei letzten Commits zurücknehmen." 
-> This says to revert the last two commits on `local` as well as the last two commits on `pushed`.
I'd propose to reword this to:
"Um diesen Level abzuschließen musst du sowohl auf `local` also auch auf `pushed` jeweils den letzten Commit zurücknehmen."

The same issue seems to be present in gl, es_ES, es_AR and pt_BR, while it is apparently correct in fr_FR and sl_SI. Unfortunately, I cannot asses the other languages…

**Comment by pcottle** [comment]
It's because the english description was changed:
https://github.com/pcottle/learnGitBranching/commit/e61a71223b3c3024d39543a76d1bf4d6f39e0440

after the translations were completed. sadly we don't have a very robust i18n system overall -- otherwise we would "trigger" a refresh of translations whenever the english source is changed.

feel free to update them! this is why I try to resist all english content updates, given that the other languages are then left with outdated versions
